contracts/test/forkMainnet/AMMWrapperWithPath/Setup.t.sol

```diff
@@ -155,21 +156,35 @@ contract TestAMMWrapperWithPath is StrategySharedSetup {
         sig = abi.encodePacked(r, s, v, bytes32(0), uint8(2));
     }
 
+    function _createSpenderPermitFromOrder(AMMLibEIP712.Order memory order) internal view returns (SpenderLibEIP712.SpendWithPermit memory takerAssetPermit) {
+        takerAssetPermit = SpenderLibEIP712.SpendWithPermit(
+            order.takerAssetAddr,
+            address(ammWrapperWithPath),
+            order.userAddr,
+            address(ammWrapperWithPath),
+            order.takerAssetAmount,
+            AMMLibEIP712._getOrderHash(order),
+            uint64(order.deadline)
+        );
```

Comment:
We can use named parameter here to help explain what are the two `address(ammWrapperWithPath)` for

Reply:
Maybe we could also apply it to the following block in `AMMWrapper.sol`?

https://github.com/consenlabs/tokenlon-contracts/blob/2619aca5e49b6de72bc9820544c81dfe226573ec/contracts/AMMWrapper.sol#L232-L240

Reply:
Does @pilagod means that Named Parameters should apply to this block of code,
even if it's not in Foundry test?

Reply:
Yap, then we could remove the comments. Can take other's opinions into account.

Reply:
OK! Let's do it!
And sorry that it need everyone to approve again after modifying this block.